mula/scheduler/connectors/services/octopoes.py

```diff
@@ -10,8 +10,15 @@ class Octopoes(HTTPService):
     name = "octopoes"
     health_endpoint = None
 
-    def __init__(self, host: str, source: str, orgs: List[Organisation]):
+    def __init__(
+        self,
+        host: str,
+        source: str,
+        orgs: List[Organisation],
+        request_timeout: int = 10,
```

Comment:
Since it's the same `timeout` variable (functionally and by name) as the parent's variable, it doesn't really make sense to rename this from `timeout` to `request_timeout`

Reply:
My though process was that when the variable would be set in the configuration a `timeout` would perhaps be not descriptive enough on what type of timeout it would be. I have no issue with changing it to timeout if that makes it more clear.

Reply:
Yes but since `Octopoes` is an `HTTPService` which has a `timeout` property then that should be implemented. Either way the property names should be the same. I think `timeout` is pretty descriptive (coming from an `HTTPService` class) and you've also perfectly documented what that property means in the (`HTTPService`) initializer.